src/algebra/big_operators/multiset.lean

```diff
@@ -82,6 +82,11 @@ lemma prod_map_mul {m : multiset ι} {f g : ι → α} :
   prod (m.map $ λ a, f a * g a) = prod (m.map f) * prod (m.map g) :=
 multiset.induction_on m (by simp) (λ a m ih, by simp [ih]; cc)
 
+@[simp]
+lemma prod_pow_eq_pow_sum {m : multiset ℕ} {a : α} :
+  (multiset.map (λ n, a^n) m).prod = a ^ (m.sum) :=
+multiset.induction_on m (by simp) (λ n m h, by simp [h,pow_add])
```

Comment:
```suggestion
@[simp]
lemma prod_map_pow {m : multiset ℕ} {a : α} :
  (multiset.map (λ n, a^n) m).prod = a ^ m.sum :=
multiset.induction_on m (by simp) (λ n m h, by simp [h,pow_add])
```
would be more consistent with the `prod_map_mul` name above.

Can you also add the following variants?
* `prod_map_zpow` for the integers
* `finset.prod_pow` and `finset.prod_zpow` (which should be provable in one line with the multiset proof.